skylark/BUILD

```diff
@@ -12,14 +12,13 @@ load(
     "stamped_plugin_xml",
 )
 
-load("@rules_java//java:defs.bzl", "java_library")
+#
+# Description: A Starlark debugging client for IntelliJ.
+#
```

Comment:
I think this comment was supposed to stay on top of the file.

Reply:
Fixed. Yes, buidifier does this is there's no empty line to the first load. In that case it considers comment to belong to the load when reordering them.